client/bundle.js

```diff
@@ -19708,6 +19708,12 @@
 	  }
 
 	  _createClass(App, [{
+	    key: 'componentDidMount',
```

Comment:
i thought we .gitignored this file. We may have to either remove it or have git stop tracking it.
to remove: git rm --cached client/bundle.js
to keep, but stop tracking changes: git update-index --assume-unchanged client/bundle.js